### ISSUE TITLE
Maintainer status and future ownership?

### ISSUE THREAD

**Issue by AB1908** [OliverBalfour/obsidian-pandoc#88]
Hi Oliver. Before we start any discussion, we want to give you a massive thanks for all the time and effort you've put into creating this plugin. A ton of people, myself included, use the plugin for our Obsidian workflows. You've kept up the spirit of Obsidian by helping us do more inside our favorite PKM tool.

We understand that not everyone has the time to maintain and respond to issues. Active development can be challenging when there are other things to attend to. You're probably well aware that there's a number of issues with some people eagerly awaiting fixes.

A few of us would love to step in and help out in whatever way we can. Although some people have offered to implement fixes, it's hard to merge them without the right permissions. On that note, we wanted to ask if you were willing to transfer ownership of the repo to either a willing member or an organization maintained by the Obsidian community. It's a difficult decision to make right away so please take your time and let us know how you feel about this. In case you wish that we take some other route like forking the repo, that's fine by us as well.

At the end of the day,  we're happy that you've done so much for the community and we want to take your work further. We hope you can let us know soon!

Link to relevant discussion: https://discord.com/channels/686053708261228577/840286264964022302/912325658657554442

**Comment by LichLord91** [comment]
Found someone who forked it and seems to be working on some commits 👀 https://github.com/BumbrT/obsidian-html-export

**Comment by AB1908** [comment]
That's actually pretty cool. I think we can use that fork as the main one for now but we need a way to indicate that that fork has some fixes and request that folks open their issues there as well. I'll try and bring this back to the community. I never got around to implementing fixes myself sadly as EOY is quite busy for me.

**Comment by RainCat1998** [comment]
> That's actually pretty cool. I think we can use that fork as the main one for now but we need a way to indicate that that fork has some fixes and request that folks open their issues there as well. I'll try and bring this back to the community. I never got around to implementing fixes myself sadly as EOY is quite busy for me.

I also fixed some bugs and create a modified version of obsidian-pandoc. (Because I update to the latest plugin toolchain, I just create a brand new repo.) https://github.com/RainCat1998/obsidian-pandoc
You can export md note to html perfectly using my patched version.

**Comment by RainCat1998** [comment]
> That's actually pretty cool. I think we can use that fork as the main one for now but we need a way to indicate that that fork has some fixes and request that folks open their issues there as well. I'll try and bring this back to the community. I never got around to implementing fixes myself sadly as EOY is quite busy for me.

If there is an 'official' community fork, I will glad to summit my patch.

**Comment by ReaderGuy42** [comment]
Hi @AB1908, thanks!

**Comment by phillipjohnston** [comment]
I have a fork that adds proper support for exporting files with embeds of blocks and headings - if there is a new effort, I will merge my fixes in there. I'm not a JS dev so I don't want to take ownership :).

**Comment by ReaderGuy42** [comment]
> I have a fork that adds proper support for exporting files with embeds of blocks and headings - if there is a new effort, I will merge my fixes in there. I'm not a JS dev so I don't want to take ownership :).

That'd be cool. A while ago I tried installing something like that and it was over my head, so I'm going to wait on installing your fork, because I don't understand npm. But keep me up to date if this gets put in with the Community plugins :) Thanks!

**Comment by AB1908** [comment]
Ah unfortunately both [@argenos](https://github.com/argenos/) and I have been busy and we haven't been able to catch up on this. Ultimately, the idea is that we maintain the official fork as part of the community org and ask contributors to submit patches there. As of now, I suppose [BRAT](https://github.com/TfTHacker/obsidian42-brat) could be used as an alternative.

**Comment by ReaderGuy42** [comment]
No worries :) It's not that hard just to use pandoc on its own for now to convert a .md to a .docx, for example. I actually also just tried BRAT for @phillipjohnston's fork but it told me the repo was incomplete, I think because a `main.js` is missing. 
Thanks :)

**Comment by phillipjohnston** [comment]
I just uploaded the files @ReaderGuy42. If BRAT still doesn't work email me (address in profile) so we don't clutter this thread.

**Comment by ReaderGuy42** [comment]
> I just uploaded the files @ReaderGuy42. If BRAT still doesn't work email me (address in profile) so we don't clutter this thread.

Works perfectly, awesome! Thanks :)

**Comment by AB1908** [comment]
@argenos, can you fork @phillipjohnston's fork?

**Comment by argenos** [comment]
Might be best if @phillipjohnston makes a PR to master in https://github.com/obsidian-community/obsidian-pandoc/ <https://github.com/obsidian-community/obsidian-pandoc/>.

**Comment by phillipjohnston** [comment]
Happy to.
